### DISCUSSION TITLE
Squishbox not booting up.

### DISCUSSION THREAD

**Discussion by garamoniker** [GeekFunkLabs/fluidpatcher#61] [General]
Okay! So, I took the Pi out of my Squishbox the other day to try to install a wah plugin.  (As discussed in another thread I ended up not doing this.)

Now, as I've put the Pi back in the Squishbox, it seems it won't start up at all - just displaying the "LCD loading" screen.

![SquishBrick](https://user-images.githubusercontent.com/108605251/190834906-87ac07c1-1d31-44e2-a420-6fe0ce44af48.jpg)

If you could point me in the right direction of what I'm doing wrong, I would be ever so grateful. See picture attached.

**Comment by garamoniker** [comment]
Hey guys. Disregard. I did a bunch of error checking. It looks like I bricked the flash card somehow. I'm thinking it might have something to do with the adapter I bought (not the typical one included with the Squishbox).

**Reply by garamoniker** [reply]
As soon as I put in a new flash, performed a new install, we're back in business.

**Reply by albedozero** [reply]
Oof sorry to hear about that! Power supply could be an issue, but for me the thing that has caused the most headache is cheap, off-brand SD cards. This would usually show up as the Pi responding very slowly at times, taking forever to update or respond over the network, throwing lots of system errors if Iogged in at the console, and eventually just failing.
